src/wp-includes/class-wp-theme-json.php

```diff
@@ -0,0 +1,374 @@
+<?php
```

Comment:
This file comes from https://github.com/WordPress/gutenberg/blob/trunk/lib/class-wp-theme-json.php and I've removed the bits that weren't necessary just yet (processing styles, etc).